NTO/Auth/entities/Person.ttl

```diff
@@ -0,0 +1,36 @@
+@prefix ogit.Auth:              <http://www.purl.org/ogit/Auth/> .
+@prefix ogit:                   <http://www.purl.org/ogit/> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit.Auth:Person
```

Comment:
What's the relationship between an `ogit/Auth/Person` and an `ogit/Person` going to be?

Reply:
I would actually challenge both Auth/Person and Auth/Role as being duplicates of the more generic ogit/Person and ogit/Role... I would definitly advise against creating these a second time here